browser/typescript/v2/react-app/src/App.tsx

```diff
@@ -0,0 +1,198 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+import { DefaultOptions, EventWithOptionalProperties, ampli } from './ampli';
+
+const { REACT_APP_AMPLITUDE_API_KEY = '', REACT_APP_SEGMENT_WRITE_KEY = '' } = process.env;
+
+const userId = 'ampli-browser-ts-user-id';
+
+/**
+ * Start by calling ampli.load()
+ *
+ * 'ampli' is the default instance of Ampli()
+ */
+
+/**
+ * When you pull your tracking plan you can use the defaults and call load() without arguments
+ *
+ * This requires connecting your account via `ampli pull`
+ * which will set you API key in the generated Ampli SDK
+ */
+// ampli.load();
+
+/**
+ * OR Specify an environment
+ */
+// ampli.load({ environment: 'development' })
+
+/** OR Provide a specific Amplitude API key */
+// ampli.load({ client: { apiKey: REACT_APP_AMPLITUDE_API_KEY } })
+
+/**
+ * OR Use an existing Amplitude instance
+ * requires "import amplitude from 'amplitude-js';"
+ */
+// const instance = amplitude.getInstance();
+// instance.init(REACT_APP_AMPLITUDE_API_KEY, undefined, { ...DefaultOptions, logLevel: "INFO" });
+// ampli.load({ client: { instance } });
+
+/**
+ * OR Use an existing window.amplitude instance from code snippet (not recommended)
+ */
+// const instance = window.amplitude.getInstance().init(REACT_APP_AMPLITUDE_API_KEY);
+// ampli.load({ client: { instance: instance as any } });
+
+/**
+ * OR Specify AmplitudeClient 'options'
+ */
+ampli.load({
+  client: {
+    apiKey: REACT_APP_AMPLITUDE_API_KEY,
+    options: { ...DefaultOptions, logLevel: 3 },
+  },
+});
+
+/**
+ * For testing you can disable ampli
+ */
+// ampli.load({
+//   disabled: process.env.IS_TESTING ? true : false,
+// });
+
+/**
+ * Make as many Ampli instances as you want
+ */
+// const ampli2 = new Ampli();
+// ampli2.load({ client: { apiKey: 'api-key-2' } });
+
+/**
+ * 3rd party destination support
+ */
+// const segmentMiddleware = getSegmentMiddleware(REACT_APP_SEGMENT_WRITE_KEY);
```

Comment:
Please remove the middleware example.
Could you also add a segment plugin example?

Reply:
Does Kotlin have any plugin example?

Reply:
The example is in this pr https://github.com/amplitude/ampli-examples/pull/115. 
You can also do this in the follow up pr.